packages/dappmanager/src/calls/httpsPortal.ts

```diff
@@ -71,6 +76,37 @@ export async function httpsPortalExposableServicesGet(): Promise<
   });
 }
 
+/**
+ * HTTPs Portal: set env value LOCAL_PROXYING
+ * - LOCAL_PROXYING defines in the https to expose or not port 80 to my.dappnode.local (avahi).
+ * - Allows/restricts access to the UI
+ * - HTTPs PR: https://github.com/dappnode/DNP_HTTPS/pull/51
+ */
+export async function httpsLocalProxyingEnableDisable(
+  enable: HttpsLocalProxyingStatus
+): Promise<void> {
+  await httpsPortal.localProxyingEnableDisable(enable);
+}
+
+/** HTTPs Portal: get env value LOCAL_PROXYING */
+export async function httpsLocalProxyingGet(): Promise<
+  HttpsLocalProxyingStatus
+> {
+  const composeHttps = new ComposeFileEditor(params.HTTPS_PORTAL_DNPNAME, true);
+  const httpsService = composeHttps.services()[params.HTTPS_PORTAL_DNPNAME];
+  const httpsEnv = httpsService.getEnvs();
+  if (httpsEnv[params.HTTPS_PORTAL_LOCAL_PROXYING_ENVNAME] === undefined)
+    throw Error(
+      `${params.HTTPS_PORTAL_LOCAL_PROXYING_ENVNAME} does not exist on compose file`
+    );
+  const localProxying = httpsEnv[params.HTTPS_PORTAL_LOCAL_PROXYING_ENVNAME];
+  if (localProxying !== "true" && localProxying !== "false")
+    throw Error(
+      `Invalid value of ${params.HTTPS_PORTAL_LOCAL_PROXYING_ENVNAME}`
+    );
```

Comment:
You should not throw errors here if the value is undefined or not true or false. Please check the logic on the HTTPs side and recreate it here. Add a link as a comment to the exact logic. I think it something like:
```ts
isEnabled = /true/i.test(LOCAL_PROXY)
```